backend/kernelCI/settings.py

```diff
@@ -48,6 +48,13 @@ def get_json_env_var(name, default):
 if isBooleanOrStringTrue(ENV_DEBUG):
     DEBUG = True
 
+DEBUG_SQL_QUERY = False
+# This is a dedicated debug env for SQL queries, it is not combine
+# with debug because it outputs too much data,
+ENV_DEBUG_SQL_QUERY = get_json_env_var("DEBUG_SQL_QUERY", False)
```

Comment:
What do you think about enabling `ENV_DEBUG_SQL_QUERY` only if `DEBUG` is also true?